dmc/projects/use_cases/create.rb

```diff
@@ -18,6 +18,7 @@ def run!
 
       ProjectRepository.save project
       add_environment! project
+      JobRunner.perform CloneJob, project
```

Comment:
You should use an `id` or a `hash` instead of the `project` object. We don't know when `Sidekiq` will perform the job, maybe the `project` could have changed since it has been enqueued.